pants.toml

```diff
@@ -49,6 +49,7 @@ inits = true
 
 [black]
 version = "black==20.8b1"
+extra_requirements = ["regex>=2021.3.17"]
```

Comment:
Why do we need this?

Reply:
this is the config we have in our internal repo... it works.... If I really think about it was probably added to override the pants default setting here, which I think is setuptools (or at least used to be) which caused issues in the past.

Reply:
I double checked, there is no longer a need for this (at least with recent pants versions)